rdr_service/dao/biobank_order_dao.py

```diff
@@ -824,6 +824,11 @@ def _do_update_with_patch(self, session, order, resource, expected_version):
             order.restoredSiteId = get_site(resource["restoredInfo"])
             order.restoredTime = clock.CLOCK.now()
             order.orderStatus = BiobankOrderStatus.UNSET
+        elif resource["status"].lower() == "re-pairing":
```

Comment:
this is cool. I'll need to learn more about the resource['status'] and how it works with re-pairing.